src/Type/Enum/EnumCaseObjectType.php

```diff
@@ -43,9 +47,11 @@ public function getEnumCaseName(): string
 
 	public function describe(VerbosityLevel $level): string
 	{
-		$parent = parent::describe($level);
-
-		return sprintf('%s::%s', $parent, $this->enumCaseName);
+		$value = $level->getLevelValue();
+		if (!array_key_exists($value, $this->descriptions)) {
+			$this->descriptions[$value] = sprintf('%s::%s', parent::describe($level), $this->enumCaseName);
+		}
+		return $this->descriptions[$value];
```

Comment:
Wouldn't it make sense to cache this in the parent instead? In ObjectType

Reply:
it also works. it makes the repro 0,2-0,3 seconds slower though